crates/agentgateway/src/proxy/httpproxy.rs

```diff
@@ -472,6 +472,12 @@ impl HTTPProxy {
 		.ok_or(ProxyError::RouteNotFound)?;
 		log.route_rule_name = selected_route.rule_name.clone();
 		log.route_name = Some(selected_route.route_name.clone());
+		// Record the matched path for tracing/logging span names
+		log.path_match = Some(match &path_match {
+			crate::types::agent::PathMatch::Exact(p) => p.to_string(),
+			crate::types::agent::PathMatch::PathPrefix(p) => format!("{}*", p),
```

Comment:
nit:
```suggestion
			crate::types::agent::PathMatch::PathPrefix(p) => format!("{}/*", p),
```

wdyt?

For example, if you have `pathPrefix: /foo`, we would NOT match `/foobar` so I think `/foo/*` may be more clear? (not 100%, since `/foo` is also valid, but this is not a regex just a human descriptor so its fine?)

Reply:
Yeah that actually makes more sense- I think we'd want to match only `/foo/bar` not `/foobar`